frontend/public/components/utils/webhooks.tsx

```diff
@@ -90,9 +95,9 @@ export const WebhookTriggers: React.FC<WebhookTriggersProps> = (props) => {
       setWebhookSecrets(_.compact(secrets));
       setLoaded(true);
     });
-  }, [isLoaded, canGetSecret]);
+  }, [webhookTriggers, isLoaded, canGetSecret, namespace]);
```

Comment:
I believe this is going to re-fetch all the secrets on any build config change (not just on changes to triggers).

Reply:
@spadgett tested this use-case and BC1 was not re-fetched if I updated BC2 (with adding trigger or build strategy parameters)

Reply:
I mean any change to the current build config. For instance, I believe it will fetch all secrets again if, say, an annotation changes.